src/helics/apps/Player.cpp

```diff
@@ -1,857 +1,863 @@
-    }
-    else
-    {
-        auto kp = endpointName.find_first_of ("./");
-        if (kp == std::string::npos)
-        {
-            endpoints.emplace_back (fed.get (), endpointName, endpointType);
-        }
-        else
-        {
-            endpoints.emplace_back (GLOBAL, fed.get (), endpointName, endpointType);
-        }
-    }
-    eptids[endpointName] = static_cast<int> (endpoints.size ()) - 1;
-}
-
-int Player::loadArguments (boost::program_options::variables_map &vm_map)
-{
-    App::loadArguments (vm_map);
-    if (vm_map.count ("datatype") > 0)
-    {
-        defType = helics::getTypeFromString (vm_map["datatype"].as<std::string> ());
-        if (defType == helics::helics_type_t::helicsInvalid)
-        {
-            std::cerr << vm_map["datatype"].as<std::string> () << " is not recognized as a valid type \n";
-            return -3;
-        }
-    }
-    if (vm_map.count ("timeunits")>0)
-    {
-        try
-        {
-            units = timeUnitsFromString (vm_map["timeunits"].as<std::string> ());
-            timeMultiplier = toSecondMultiplier (units);
-        }
-        catch (...)
-        {
-            std::cerr << vm_map["timeunits"].as<std::string> () << " is not recognized as a valid unit of time \n";
-        }
-    }
-    return 0;
-}
-
-}  // namespace apps
-}  // namespace helics
-
-static int hasB64Wrapper (const std::string &str)
-{
-    if (str.front() == '\"')
-    {
-        if ((str.compare(2, 3, "64[") == 0) && (str[str.size()-2] == ']'))
-        {
-            return 5;
-        }
-        if ((str.compare(5, 3, "64[") == 0) && (str[str.size() - 2] == ']'))
-        {
-            return 8;
-        }
-    }
-    else
-    {
-        if ((str.compare(1, 3, "64[") == 0) && (str.back() == ']'))
-        {
-            return 4;
-        }
-        if ((str.compare(4, 3, "64[") == 0) && (str.back() == ']'))
-        {
-            return 7;
-        }
-    }
-    
-    return 0;
-}
-
-static std::string decode (std::string &&stringToDecode)
-{
-    if (stringToDecode.empty ())
-    {
-        return std::string ();
-    }
-    auto offset = hasB64Wrapper (stringToDecode);
-    if (offset != 0)
-    {
-        if (stringToDecode.back() == '\"')
-        {
-            stringToDecode.pop_back();
-        }
-
-        stringToDecode.pop_back ();
-        return utilities::base64_decode_to_string (stringToDecode, offset);
-    }
-
-    if ((stringToDecode.front () == '"') || (stringToDecode.front () == '\''))
-    {
-        return stringOps::removeQuotes (stringToDecode);
-    }
-    // move is required since you are returning the rvalue and we want to move from the rvalue input
-    return std::move (stringToDecode);
-}
+/*
+Copyright © 2017-2018,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
+All rights reserved. See LICENSE file and DISCLAIMER for more details.
+*/
```

Comment:
Why is every line in this file changed? Could the encoding of file have changed here when you made changes to the file?

Reply:
MIght have changed the line endings, when I did a clang-format on it?